src/components/CodeBlock.tsx

```
@@ -0,0 +1,68 @@
+      /(\/\/.*)/g,
+      '<span class="text-gray-500">$1</span>'
+    );
+
+    // Function calls and definitions
+    line = line.replace(
+      /(\w+)(\s*\()/g,
+      '<span class="text-yellow-300">$1</span>$2'
+    );
+  }
+  
+  return <span dangerouslySetInnerHTML={{ __html: line }} />;
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)